frontend/src/pages/WorkspaceLocked/WorkspaceLocked.styles.scss

```diff
@@ -1,16 +1,165 @@
-.workspace-locked-container {
-	text-align: center;
-	padding: 48px;
-	margin: 24px;
+$workspaceLocked: 'workspace-locked';
+$light-theme: 'lightMode';
```

Comment:
btw why are we using variables for these ?

Reply:
@vikrantgupta25, these best practices will help you maintain the code more easily and reduce repetition. SCSS offers several concepts to enhance the authoring experience. I’ve used DRY principles and modular CSS. You can get the vibes by reading more about these methods [here](https://sass-lang.com/documentation/variables/#:~:text=CSS%20variables%20can%20have%20different,use%20will%20stay%20the%20same.) and [here](https://www.stubbornella.org/2010/06/25/the-media-object-saves-hundreds-of-lines-of-code/).